packages/integration-tests/src/test-projects/conversation-handler/amplify/custom-conversation-handler/custom_handler.ts

```diff
@@ -21,7 +21,7 @@ const thermometer = createExecutableTool(
   },
   (input) => {
     const city = input.city;
-    if (city === 'Seattle' || city === 'Boston' || city === 'Miami') {
+    if (city === 'Seattle' || city === 'Boston') {
```

Comment:
This is unrelated to rest of PR.

I removed one of the cities because:
1. two is enough to prove that multiple tools work in single turn
2. having 3 tools makes Bedrock hallucinate empty answer more frequently causing test instabilities.